src/lib.rs

```diff
@@ -21,6 +21,11 @@
 //! }
 //! ```
 
+#[macro_use(lazy_static)]
+extern crate lazy_static;
```

Comment:
If I have it correctly then `extern crate` is no longer needed from the 2018 edition of Rust.

Reply:
Thanks. I wasn't aware of that so have made changes for how macros are imported with Rust 2018.